pybossa/importers/csv.py

```diff
@@ -49,80 +49,98 @@ def _get_data_url(self):
         """Get data from URL."""
         return self.url
 
+    def _convert_row_to_task_data(self, row, row_number):
+        task_data = {"info": {}}
+        private_fields = dict()
+        private_gold_answers = dict()
+        for idx, cell in enumerate(row):
+            if idx in self.field_header_index:
+                if self._headers[idx] == 'user_pref':
+                    if len(cell) > 0:
+                        task_data[self._headers[idx]] = json.loads(cell.lower())
+                    else:
+                        task_data[self._headers[idx]] = {}
+                else:
+                    task_data[self._headers[idx]] = cell
+            elif self._headers[idx].endswith('_gold'):
+                if cell:
+                    if self._headers[idx].endswith('_priv_gold'):
```

Comment:
Should we treat all `_gold` as private when `data_access` is enabled?

Reply:
as user mentioned _gold, we could treat it as public.